libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -100,8 +100,57 @@ void AP_Scripting::init(void) {
     }
 }
 
+MAV_RESULT AP_Scripting::handle_command_int_packet(const mavlink_command_int_t &packet) {
```

Comment:
nitpick: normally brackets are on the following line.. obviously not a blocker just mentioning it